src/sentry/api/urls.py

```diff
@@ -1963,6 +1964,11 @@
         ProjectReleaseStatsEndpoint.as_view(),
         name="sentry-api-0-project-release-stats",
     ),
+    url(
+        r"^(?P<organization_slug>[^\/]+)/(?P<project_slug>[^\/]+)/artifact-bundle/(?P<bundle_id>[^/]+)/files/$",
```

Comment:
I'd double check that this is the url we want to use. All urls in this kind of shape are using plurals if they allow for specifying an ID. eg.

```
/org/projects/releases/<release-id>/files/
/files/dsyms/
/files/source-maps/
/files/artifact-bundles/
/files/difs/
```